domain/src/main/java/com/architectcoders/aacboard/domain/use_case/search/SearchPictogramsUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.architectcoders.aacboard.domain.use_case.search
+
+import com.architectcoders.aacboard.domain.data.Response
+import com.architectcoders.aacboard.domain.data.cell.CellPictogram
+import com.architectcoders.aacboard.domain.repository.PictogramsRepository
+
+
+class SearchPictogramsUseCase(
+    private val repository: PictogramsRepository
+): suspend (String, String) -> Response<List<CellPictogram>> {
+
+    override suspend fun invoke(language: String, searchString: String): Response<List<CellPictogram>> {
+        return repository.searchPictograms(language, searchString)
+    }
+}
```

Comment:
As mentioned before, I would get the region inside the searchPictogramUseCase

Reply:
Open for discussion.